lib/omnibus/software.rb

```diff
@@ -242,6 +242,28 @@ def dependency(val)
     end
     expose :dependency
 
+    #
+    # Add a debug package path.
+    #
+    # Paths added here will be excluded from the main package and added to
+    # the debug variant instead.
+    #
+    # @example
+    #   debug_path 'foo/bar'
+    #   dependency 'quz'
+    #
+    # @param [String] val
+    #   the pathi to include in the debug build
```

Comment:
The `pathi` is back for more
```suggestion
    #   the path to include in the debug build
```